modules/nw-networkpolicy-audit-configure.adoc

```diff
@@ -0,0 +1,228 @@
+[id="nw-networkpolicy-audit-configure_{context}"]
+= Configuring network policy auditing
+
+As a cluster administrator, you can customize network policy audit logging to suit your environment.
```

Comment:
One nit, one sugg.

Nit: with "suit my environment," I wasn't clear if that meant to meet my organizations network observability needs or that the procedure would accommodate cluster and hardware specific peculiarities.  Or maybe just slice out "to suit your environment."

The sugg: Because I was initially confused by the similarity of the verification for this procedure and the "Enabling network policy audit logging", my sugg is to add cluster to this title--maybe "Configuring network policy auditing for a cluster". Maybe also add "namespace" to the other title like "Enabling network policy audit logging for a namespace".

Reply:
@mikemckiernan, brilliant, thanks!